components/helper/StatistikItem.tsx

```diff
@@ -0,0 +1,14 @@
+type StatistikItemProps = {
+  title: string;
+  times: number | string;
+};
+
+const StatistikItem = ({ title, times }: StatistikItemProps) => {
```

Comment:
Maybe we should rename this to `StatisticsItem` later on to match the English naming convention of the rest of the app.